network/v1/types.go

```diff
@@ -136,8 +189,10 @@ const (
 
 // EgressNetworkPolicyPeer specifies a target to apply egress network policy to
 type EgressNetworkPolicyPeer struct {
+	// +kubebuilder:validation:Pattern=`^(([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])/([0-9]|[12][0-9]|3[0-2])$`
```

Comment:
there a couple of OpenAPI formats supported for CRDs. Compare https://github.com/kubernetes/apiextensions-apiserver/blob/master/pkg/apis/apiextensions/v1/types_jsonschema.go#L35. CIDRs are among them.

Reply:
Will this break backward compatibility?

Reply:
It looks like you can't specify "ipv4-only cidr". (And this is only used by openshift-sdn which is not going to support ipv6.) So we should stick with the regexp. Maybe with a comment.

Reply:
If the format does not fit, the regex is fine.

Reply:
added a comment in the types.go around this.